src/ontology/foodon-edit.owl

```diff
@@ -61399,13 +61419,14 @@ AnnotationAssertion(obo:IAO_0000115 obo:FOODON_03510023 "Renamed from *FOOD FOR
 These descriptors are used either for foods intended for special dietary use as defined in 21 CFR 105 or for foods that have special characteristics indicated in the name or labeling. Such claims would include 'low calorie', 'low cholesterol', 'diet' or 'dietetic', etc. Prior to february 1992, this category of descriptors was limited solely to food for adult humans. The category is now used in conjunction with any appropriate *CONSUMER GROUP* descriptor(s), for example, 'low sugar baby food' would be indexed by *INFANT OR TODDLER FOOD* as well as by *LOW SUGARS FOOD*."@en)
 AnnotationAssertion(obo:IAO_0000412 obo:FOODON_03510023 "http://langual.org")
 AnnotationAssertion(oboInOwl:hasDbXref obo:FOODON_03510023 "http://www.langual.org/langual_thesaurus.asp?termid=P0023")
+AnnotationAssertion(oboInOwl:hasDbXref obo:FOODON_03510023 "http://www.langual.org/langual_thesaurus.asp?termid=P0032")
```

Comment:
Is this a stray duplicate of line above?  Sometimes that can happen when editing with Protege.

Reply:
This is actually a genuine difference, just two very similar IDs (P0023 and P0032). The langual term for label claim/dietary use, only seemed to pertain to dietary requirements so Rhi and I thought it may be worth including on the dietary claim or use [FOODON:03510023] term.